src/main/java/co/com/bancolombia/task/GenerateAcceptanceTestTask.java

```diff
@@ -1,18 +1,28 @@
 package co.com.bancolombia.task;
 
 import co.com.bancolombia.exceptions.CleanException;
+import co.com.bancolombia.factory.ModuleFactory;
 import co.com.bancolombia.factory.tests.ModuleFactoryTests;
 import co.com.bancolombia.utils.Utils;
 import java.io.IOException;
 import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.options.Option;
 
 public class GenerateAcceptanceTestTask extends CleanArchitectureDefaultTask {
+  private String name;
+
+  @Option(option = "name", description = "Set driven adapter name")
+  public void setName(String name) {
+    this.name = name;
+  }
```

Comment:
it would be more usable if this parameter were optional.

Reply:
Ok, following your recommendation, now the parameter name in gat task is optional